db/schema.rb

```diff
@@ -19,11 +19,12 @@
   end
 
   create_table "participants", force: :cascade do |t|
-    t.integer "project_id", null: false
-    t.string  "email",      null: false
+    t.integer  "project_id",     null: false
+    t.string   "email",          null: false
+    t.datetime "last_responded"
   end
 
-  add_index "participants", ["project_id", "email"], name: "index_participants_on_project_id_and_email", unique: true
+  add_index "participants", ["project_id", "email", "last_responded"], name: "index_participants_on_project_id_and_email_and_last_responded", unique: true
```

Comment:
I don't think we need to index on last_responded, unless we need project_id, email, and last_responded to be unique?

The point of adding indexing was to ensure uniqueness for project_id and email, so that we don't have multiple of the same participants, although perhaps that could be ensured via a validations in the Model

Reply:
You're right. Gonna remove it!

Reply:
Thanks!